other_scripts/reporting/ams_summary.py

```diff
@@ -0,0 +1,50 @@
+##############################################
+# For a given pair of integer indices,
+# print a table summarizing the annual maximum series
+# data at that point.
+##############################################
+
+import sys, os
+import xarray as xr
+
+DATADIR     = "/workspace/Shared/Tech_Projects/DOT/project_data/wrf_pcpt/annual_maximum_series/"
+
+DURATIONS   = [ '60m','2h','3h','6h','12h','24h','3d','4d','7d','10d','20d','30d','45d','60d' ]
```

Comment:
Could we add the 2d duration to this and the relevant "other" scripts? e.g. point_data.py, point_data_general.py